### PR TITLE
Sadqan paraqlitos for esap046

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

**Comment by eu-genia** [comment]
@thea-m sorry again some branch confusion (have a very slow connection or laptop, don't know which, but GitHub desktop reacts too slowly), the first two submits were already in another branch, the last two are new

File: 1001-2000/LIT1491Gadlas.xml [hunk 1]
```diff
@@ -56,10 +60,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+            
+               <note>The embedded colophon states that the work was completed in the month of Śane,
+                  <origDate when="1523">year of mercy 175 (= 1523 A.D)</origDate>, in the time of King Lǝbna Dǝngǝl, Metropolitan Marqos (d.
+                  1529-30), Archbishop John XIII of Alexandria and governor of ʿAgame (śǝyuma ʿagāme)
+                  Natan.</note>
+               <note>Furthermore, it says that Dānʾel “the falāsi” (“the stranger, the
+                  wanderer”) is the one who has “translated” (za-ʾaʿlawkǝwwa) the work (Gadla
+                  and Taʾāmra samāʿtāta Ṗarāqliṭos), who is at the same time the one who has
+                  written this colophon (ወዘአዕለውክዋ፡ አነ፡ ዳንኤል፡ ፈላሲ፨ “I myself, Dānʾel the
+                  stranger, am the one who have compiled it”). It says, besides, that Tasfā
```
Comment (thea-m) [inline 1001-2000/LIT1491Gadlas.xml:91]: who has?

File: new/LIT5877DossierParaqlitos.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5877DossierParaqlitos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Hagiographic Dossier of the Martyrs of Ṗarāqliṭos</title>
```
Comment (thea-m) [inline new/LIT5877DossierParaqlitos.xml:8]: en

File: new/LIT5877DossierParaqlitos.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <ab>The collection consisiting Life, Miracles, and Hymns in honour of the Martyrs of Ṗarāqliṭos</ab>
```
Comment (thea-m) [inline new/LIT5877DossierParaqlitos.xml:33]: consisting of the....
Could you maybe mark up the texts here in with `<ref type="work" corresp="...">`? It has shown that this makes things a lot easiert for many users.

File: new/LIT5877DossierParaqlitos.xml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            <change who="ES" when="2020-04-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:P129_is_about" active="LIT5877DossierParaqlitos" passive="PRS10773SadeqP"/>
+                    <relation name="saws:contains" active="LIT5877DossierParaqlitos" passive="LIT1491Gadlas">
+                        <desc>The dossier contains the Acts of the Righteous of of Ṗarāqliṭos</desc>
```
Comment (thea-m) [inline new/LIT5877DossierParaqlitos.xml:56]: of of in all relations

File: new/LIT5878MiraclesParaqlitos.xml [hunk 5]
```diff
@@ -0,0 +1,147 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5878MiraclesParaqlitos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+               
+                <title xml:lang="gez" xml:id="t1">ተአምረ፡ ሰማዕታተ፡ ጰራቅሊጦስ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Taʾāmra samāʿtāta Ṗarāqliṭos</title>
```
Comment (thea-m) [inline new/LIT5878MiraclesParaqlitos.xml:10]: Taʾammǝra

File: new/LIT5878MiraclesParaqlitos.xml [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        <revisionDesc>
+            <change who="ES" when="2020-04-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>Text and Miracles provided after <ref type="mss" target="ESap046"/></note>
+                <div type="textpart" subtype="miracle" xml:id="Miracle1"><label>Miracle of the divine punishment (birds drinking human being’s
+                    brains)</label>
```
Comment (thea-m) [inline new/LIT5878MiraclesParaqlitos.xml:59]: beings'? Seems to be collective

File: new/LIT5877DossierParaqlitos.xml [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <ab>The collection consisiting Life, Miracles, and Hymns in honour of the Martyrs of Ṗarāqliṭos</ab>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
```
Comment (thea-m) [inline new/LIT5877DossierParaqlitos.xml:38]: Poetry?

**Comment by thea-m** [comment]
No worries!

File: 1001-2000/LIT1491Gadlas.xml (hunk 1 above)
Comment (eu-genia) [reply 1001-2000/LIT1491Gadlas.xml]: ok

File: new/LIT5877DossierParaqlitos.xml (hunk 2 above)
Comment (eu-genia) [reply new/LIT5877DossierParaqlitos.xml]: sure

File: new/LIT5877DossierParaqlitos.xml (hunk 3 above)
Comment (eu-genia) [reply new/LIT5877DossierParaqlitos.xml:33]: ok

File: new/LIT5878MiraclesParaqlitos.xml (hunk 5 above)
Comment (eu-genia) [reply new/LIT5878MiraclesParaqlitos.xml:10]: sure

File: new/LIT5878MiraclesParaqlitos.xml (hunk 6 above)
Comment (eu-genia) [reply new/LIT5878MiraclesParaqlitos.xml:59]: could be

**Comment by eu-genia** [comment]
Hope all OK

**Pull request merged**